_posts/2025-03-21-release.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Release 2025.12.1
+author: Priya
+date: 2025-03-21
+category: Jekyll
+layout: post
+---
+### Product Enhancement :
+* Data Settings: Added a dropdown in the Data Settings modal to allow users to select a quote character during data ingestion.
+* Intent-Based Text Extraction: Enhanced the Extract Text rule to enable users to specify their intent for text extraction in plain English (e.g., "find all email addresses"), which is then automatically translated into the corresponding Regex pattern.
+* Mammoth new version - The latest version brings an improved user experience with a redesigned interface. Users can now access and manage everything from a single page, streamlining workflows and reducing the need to switch between different sections.
+The new UI is now live as alpha release. If you’re interested in early access, reach out to us at [support@mammoth.io](mailto:support@mammoth.io) or via the in-app chat.
```

Comment:
The new UI is now live as alpha release available to use via a toggle in the settings page. This will be released to 100% users in a few weeks. If you’re interested in early access, or have any feedback reach out to us at [support@mammoth.io](mailto:support@mammoth.io) or via the in-app chat.